webpack/assets/javascripts/react_app/components/ModelsTable/ModelsTable.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import URI from 'urijs';
+import { Spinner } from 'patternfly-react';
+import Table from '../common/table';
+import { STATUS } from '../../constants';
+import './ModelsTable.css';
+
+class ModelsTable extends React.Component {
+  componentDidMount() {
+    this.props.updateSearch(this.props.data);
```

Comment:
Could you destruct `data` and create a new object?
I feel like it's risky to pass the data object as is.